### PR TITLE
debian/rules: strip ceph-base libraries

### PR DESCRIPTION
**Pull request by liewegas**
This includes rados classes and ec plugins.
http://tracker.ceph.com/issues/22640

### CODE REVIEW

**Comment by liewegas** [comment]
If this is the right thing, I think we also need to add -dbg packges for python-{ceph,rbd,rados}[3] too

**Comment by liewegas** [comment]
@tchaikov updated

File: debian/control [hunk 1]
```diff
@@ -859,6 +949,21 @@ Description: Python 2 libraries for the Ceph libcephfs library
  This package contains Python 2 libraries for interacting with Ceph's
  CephFS file system client library.
 
+Package: python-cephfs-dbg
+Architecture: linux-any
+Section: debug
+Depends: libcephfs2 (= ${binary:Version}),
```
Comment (ktdreyer) [inline debian/control]: Not `Depends: python-cephfs` here?

Comment (liewegas) [reply debian/control]: thanks, fixed!

File: debian/control [hunk 2]
```diff
@@ -795,6 +825,21 @@ Description: Python 2 libraries for the Ceph librbd library
  This package contains Python 2 libraries for interacting with Ceph's
  RBD block device library.
 
+Package: python-rbd-dbg
+Architecture: linux-any
+Section: debug
```
Comment (tchaikov) [inline debian/control]: should have 
```
Priority: extra
Depends: python-dbg //// python3-dbg for python3 package
```

see https://wiki.debian.org/Python/DbgBuilds and https://wiki.ubuntu.com/PyDbgBuilds

Comment (liewegas) [reply debian/control]: fixed.  also fixed some of the >= versions to = versions.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2018-01-11_05:03:22-rados-wip-kefu-testing-2018-01-11-1036-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2018-01-14_09:26:29-rados-wip-kefu-testing-2018-01-11-1036-distro-basic-mira/

failed tests are tracked by 

- http://tracker.ceph.com/issues/20236
- http://tracker.ceph.com/issues/22527
- http://tracker.ceph.com/issues/22668

**Pull request merged**